pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -565,6 +565,10 @@ spec:
           value: "$(tasks.collect-atlas-params.results.ssoTokenUrl)"
         - name: bombasticApiUrl
           value: "$(tasks.collect-atlas-params.results.bombasticApiUrl)"
+        - name: retryAWSSecretName
```

Comment:
These new parameters should be passed to upload-sbom-to-atlas and not these two tasks, right? They don't support this param.

Reply:
Oh, sorry, I didn't realize both use the same `upload-sbom-to-atlas` ref.